### PR TITLE
bug 1616536: change format of maxmind db version in /__heartbeat__

### PR DESCRIPTION
**Pull request by willkg**
The version is a seconds-since-epoch of the build. This changes it to a
`YYYY-MM-DDTHH:MM:SSZ` format which is what Telemetry uses.

### CODE REVIEW

**Comment by willkg** [comment]
@jwhitlock The thing about this is that since I implemented it as a property on the geoip class, it's easier to set the format to what Telemetry has and use that for the version in heartbeat than it is to have two different formats of the version. Thus I came around to your line of thinking and changed it.

File: ichnaea/geoip.py [hunk 1]
```diff
@@ -458,9 +459,10 @@ def age(self):
     def version(self):
         """
         :returns: The version of the database.
-        :rtype: int
+        :rtype: string as iso-8601 YYYY-MM-DDTMM:HH:SSZ
         """
-        return self.metadata().build_epoch
+        epoch = self.metadata().build_epoch
+        return datetime.datetime.fromtimestamp(epoch).isoformat() + "Z"
```
Comment (willkg) [inline ichnaea/geoip.py]: The Z thing is gross. I'll look at python-dateutil to see if there's a better way.

Comment (jwhitlock) [reply ichnaea/geoip.py]: I've had painful experience trying to manually create JS-compatible timestamps with a UTC offset, but I'm having a hard time recalling the details. This is probably close enough, but we should be sure to double-check the generated format in stage.

I was thinking of [specifying seconds](https://docs.python.org/3/library/datetime.html#datetime.datetime.isoformat), but I think we get that for free when generating from a timestamp:
```suggestion
        return datetime.datetime.fromtimestamp(epoch).isoformat(timespec="seconds") + "Z"
```

Comment (willkg) [reply ichnaea/geoip.py]: https://stackoverflow.com/questions/19654578/python-utc-datetime-objects-iso-format-doesnt-include-z-zulu-or-zero-offset

That suggests Python by itself won't do the "Z" form for iso-8601 timestamps. So I think we're stuck with "Z".

Also, the tests use a null geoip wrapper, but it's squirrely in that it doesn't actually test the `GeoipWrapper` code at all. I don't want to fix that in this PR, but it means that we have to test it by hand.

I'm going to switch the code to use `strftime` which guarantees the right format and is easier to read.

**Comment by willkg** [comment]
Thank you!
